pkgs/applications/misc/bicon/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  pkg-config,
+  perl,
+  fribidi,
+  kbd,
+  xkbutils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bicon";
+  version = "38725c062a83ab19c4e4b4bc20eb9535561aa76c";
+
+  src = fetchFromGitHub {
+    owner = "behdad";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "0hdslrci8pq300f3rrjsvl5psfrxdwyxf9g2m5g789sr049dksnq";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [ fribidi kbd xkbutils perl ];
+
+  preConfigure = ''
+    patchShebangs ./autogen.sh
+    patchShebangs ./configure
+    patchShebangs ./font/bicon-bdf2psf.pl
+  '';
+
+  configureFlags = [
+    "--enable-fribidi --with-gtk=2.0,3.0"
```

Comment:
`configureFlags` is [a list of strings passed as additional arguments](https://nixos.org/nixpkgs/manual/#ssec-configure-phase). It is probably desired to split those two flags into two discrete strings. (It ends up working due to an implementation detail.)